src/components/standalone/dns_dhcp/DynamicLeases.vue

```diff
@@ -47,30 +78,108 @@ function applyFilterToDynamicLeases(dynamicLeases: DynamicLease[], filter: strin
       dynamicLease.hostname.toLowerCase().includes(lowerCaseFilter) ||
       dynamicLease.ipaddr.toLowerCase().includes(lowerCaseFilter) ||
       dynamicLease.macaddr.toLowerCase().includes(lowerCaseFilter) ||
-      dynamicLease.interface.toLowerCase().includes(lowerCaseFilter)
+      dynamicLease.interface.toLowerCase().includes(lowerCaseFilter) ||
+      dynamicLease.device.toLowerCase().includes(lowerCaseFilter)
   )
 }
+
+const interfaceFilterOptions = computed(() => {
+  // Collect all interface values, treating empty string as 'unknown'
+  const interfaces = items.value.map((item) =>
+    item.interface && item.interface.trim() !== '' ? item.interface : 'unknown'
```

Comment:
We can simplify this to:
```suggestion
    item.interface?.trim() !== '' ? item.interface : 'unknown'
```